core/src/commonJvmMain/kotlin/com/ustadmobile/core/catalog/contenttype/DeleteContainerPlugin.kt

```diff
@@ -0,0 +1,86 @@
+package com.ustadmobile.core.catalog.contenttype
+
+import com.ustadmobile.core.account.Endpoint
+import com.ustadmobile.core.contentjob.*
+import com.ustadmobile.core.db.JobStatus
+import com.ustadmobile.core.db.UmAppDatabase
+import com.ustadmobile.core.torrent.UstadTorrentManager
+import com.ustadmobile.core.util.DiTag
+import com.ustadmobile.core.util.ext.deleteFilesForContentEntry
+import com.ustadmobile.door.DoorUri
+import com.ustadmobile.door.ext.DoorTag
+import com.ustadmobile.door.ext.toDoorUri
+import com.ustadmobile.lib.db.entities.ContainerEntryFile
+import com.ustadmobile.lib.db.entities.ContentEntryWithLanguage
+import com.ustadmobile.lib.db.entities.ContentJobItemAndContentJob
+import io.ktor.client.*
+import org.kodein.di.DI
+import org.kodein.di.direct
+import org.kodein.di.instance
+import org.kodein.di.on
+import java.io.File
+
+class DeleteContainerPlugin(
```

Comment:
This is really DeleteContentEnrtyPlugin ... this is working with a content entry now...